Test/Unit/Block/Customer/CreditCardTest.php

```diff
@@ -122,4 +131,11 @@ public function getCreditCardCollection()
         $this->assertSame($this->collectionFactoryMock, $this->block->getCreditCardCollection());
     }
 
+    /**
+     * @test
+     */
+    public function getFormKey(){
```

Comment:
(minor) please move the opening curly brace to the next line to comply with PSR2

Reply:
@oxigenka I updated the code. Can you re-check?

Reply:
@oxigenka I updated the code. Can you re-check?